deploy/podman/flightctl-alertmanager-proxy/flightctl-alertmanager-proxy-init.container

```diff
@@ -5,14 +5,13 @@ After=flightctl-alertmanager.service
 Wants=flightctl-alertmanager.service
 
 [Container]
-Image=registry.access.redhat.com/ubi9/ubi-minimal
+Image=quay.io/flightctl/flightctl-alertmanager-proxy:latest
```

Comment:
@gshilin-sdb @kkyrazis The base image changed for this service. We'll need to ensure downstream reflects this.